### DISCUSSION TITLE
Backups not working on Vapor, mysqldump command not found

### DISCUSSION THREAD

**Discussion by carlosrgzm** [spatie/laravel-backup#1169] [Q&A]
When I'm trying to use the package in Vapor, if I do a backup, I'm having this issue:

Command not found : sh: mysqldump: command not found

Any idea on how can I fix it ?

**Comment by iAmKevinMcKee** [comment]
I am having the same problem after I just migrated to Vapor.

**Reply by PaolaRuby** [reply]
#1374

**Comment by polbatllo** [comment]
Did any one try to use custom docker images in vapor for that?

**Reply by PaolaRuby** [reply]
#1374

**Comment by PaolaRuby** [comment]
Same as #1374

**Comment by erikn69** [comment]
https://docs.vapor.build/1.0/projects/environments.html#runtime

```batch
# Add the `mysql` client...
RUN apk --update add mysql-client
```
Or on `database.connections.mysql` you can add custom properties, example
```
'dump' => [
        'dump_binary_path' => '/MariaDB 10.5/bin/', // only the path, so without `mysqldump` or `pg_dump`
        'use_single_transaction',
        'timeout' => 60 * 5, // 5 minute timeout
    ]
```
But find path to `mysqldump` is a question for vapor, not for laravel-backup
